eng/Versions.props

```diff
@@ -62,15 +62,15 @@
   -->
   <PropertyGroup Label="Automated">
     <!-- Packages from dotnet/arcade -->
-    <MicrosoftDotNetGenAPIPackageVersion>1.0.0-beta.20124.2</MicrosoftDotNetGenAPIPackageVersion>
+    <MicrosoftDotNetGenAPIPackageVersion>1.0.0-beta.20113.5</MicrosoftDotNetGenAPIPackageVersion>
```

Comment:
Why aren't the Arcade and Blazor changes not automated? We still have Maestro++ subscriptions for those repos targeting this branch. Are automated updates not happening?

Reply:
I don't recall seeing Maestro updates for versions other than the Components.Runtime package. See https://github.com/dotnet/aspnetcore/commits/blazor-wasm/eng/Version.Details.xml